R/ae_forestly.R

```diff
@@ -50,6 +50,16 @@ ae_forestly <- function(outdata, filter = c("prop", "n"), width = 1400) {
     FUN.VALUE = character(1)
   )
 
+  for (par in parameters[(!(parameters %in% unique(outdata$parameter_order)))]){
+    outdata$tbl <-
+      rbind(outdata$tbl, NA) |>
+      dplyr::mutate(
```

Comment:
Hi @fukuhiro2023. We could replace use of `dplyr::mutate()` by base R function since we don't use `dplyr` for development in out `forestly` function.

Reply:
Thanks, I didn't aware of it. I have chaned the code.